libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -83,9 +83,6 @@ internal class MapboxTripSession(
                     }
                 }
             }
-        }
-        mainJobController.scope.launch {
-            updateRouteJob?.join()
             this@MapboxTripSession.route = route
         }
```

Comment:
@Zayankovsky we can't remove it. we should wait for a job completion to have the same route on SDK and NN side

Reply:
But we still wait for it. `this@MapboxTripSession.route = route` is called only after `navigator.setRoute` finishes.

Reply:
ah, I've missed a removed `}`, now it's inside a `updateRouteJob`